cmd/terramate/cli/run.go

```diff
@@ -119,6 +119,12 @@ func selectPlanFile(terraformPlan, tofuPlan string) (planfile, provisioner strin
 func (c *cli) runOnStacks() {
 	c.gitSafeguardDefaultBranchIsReachable()
 
+	if len(c.parsedArgs.Run.Command) > 0 {
```

Comment:
Can you briefly explain why this becomes necessarily now?

Reply:
they change the behavior of specifically `arg:"" passthrough:""`:
https://github.com/alecthomas/kong/pull/436

Maybe I'm missing something but I cannot make it work in in the way it was before...
The `passthrough:""` prevents kong from interpretating the flags, so it's needed otherwise commands with same flags as Terramate gets interpreted. The `arg:""` is also needed to ... make it an argument, unless there's other option to keep old behavior :(